### PR TITLE
Feature/garments sizes

### PR DESCRIPTION
**Pull request by p4pupro**
---
name: Add Garments Sizes Faker
about: new feature to use Garments Sizes. see example: XS, S, M, L, XL, XXL, XXXL
title: 'Garments Sizes'
labels: 'Garments Sizes'
assignees: ''

---

**Is your feature request related to a problem? Please describe.**
No, is just a feature.

**Describe the solution you'd like**
None.

**Describe alternatives you've considered**
None.

**Additional context**
None.

### CODE REVIEW

File: src/main/java/net/datafaker/GarmentSize.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  public String size() {
+    return faker.fakeValuesService().fetchString("garments.sizes");
+  }
+
+  /**
+   * This method returns a garment XS size
+   *
+   * @return a XS string of garment size
+   */
+  public String anXS() { return faker.fakeValuesService().fetchString("garments.xs"); }
```
Comment (bodiam) [inline src/main/java/net/datafaker/GarmentSize.java:28]: I don't really understand this. Why would you want this?

File: src/main/resources/en/garments_sizes.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+en:
+  faker:
+    garments:
+      sizes:  ["XS", "S", "M", "L", "XL", "XXL", "XXXL"]
+      xs: ["XS"]
```
Comment (bodiam) [inline src/main/resources/en/garments_sizes.yml:5]: Same for the mapping here, why would you map xs to XS?

**Comment by bodiam** [comment]
Hi @p4pupro , thanks for the PR! I've left a few comments, could you have a look? Also, it seems the build is failing, I'm not sure if that's caused by this PR, but could you please have a look at it? Thanks!

File: src/main/java/net/datafaker/GarmentSize.java (hunk 1 above)
Comment (p4pupro) [reply src/main/java/net/datafaker/GarmentSize.java:28]: I just want a `size` method to get a random size, and a specific method for each size to get a specific size.

Comment (bodiam) [reply src/main/java/net/datafaker/GarmentSize.java:28]: But if you know which size you want, why do even want to call the method?

Comment (p4pupro) [reply src/main/java/net/datafaker/GarmentSize.java]: totally true... make no sense... sorry abut that.. i did a commit to solved this.

File: src/main/resources/en/garments_sizes.yml (hunk 2 above)
Comment (p4pupro) [reply src/main/resources/en/garments_sizes.yml]: I remove senseless mapping!

File: src/main/java/net/datafaker/Faker.java [hunk 3]
```diff
@@ -647,9 +647,7 @@ public GameOfThrones gameOfThrones() {
         return getProvider(GameOfThrones.class, () -> new GameOfThrones(this));
     }
 
-    public GarmentSize garmentSize() {
-    return getProvider(GarmentSize.class, () -> new GarmentSize(this));
-  }
+    public GarmentSize garmentSize() { return getProvider(GarmentSize.class, () -> new GarmentSize(this)); }
```
Comment (panilya) [inline src/main/java/net/datafaker/Faker.java:650]: I think multi-line would be better in order to maintain codestyle

Comment (p4pupro) [reply src/main/java/net/datafaker/Faker.java:650]: okey, but you have many examples with one line

Comment (panilya) [reply src/main/java/net/datafaker/Faker.java]: @p4pupro No, IDE collapses code to make classes more readable

Comment (p4pupro) [reply src/main/java/net/datafaker/Faker.java]: Okey, I did the change to multi line.

**Pull request merged**